services/user.py

```diff
@@ -0,0 +1,62 @@
+from typing import Optional
+
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ObjectDoesNotExist
+
+User = get_user_model()
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: Optional[str] = None,
+        first_name: Optional[str] = None,
+        last_name: Optional[str] = None
+) -> User:
+    user = User.objects.create_user(username=username, password=password)
+
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
+    return user
+
+
+def get_user(user_id: int) -> Optional[User]:
+    try:
+        user = User.objects.get(id=user_id)
+        return user
+    except ObjectDoesNotExist:
+        return None
+
+
+def update_user(
+        user_id: int,
+        username: Optional[str] = None,
+        password: Optional[str] = None,
+        email: Optional[str] = None,
+        first_name: Optional[str] = None,
+        last_name: Optional[str] = None
+) -> User:
+    try:
+        user = User.objects.get(id=user_id)
+
+        if username:
+            user.username = username
```

Comment:
When updating the username, ensure that the new username is unique to prevent integrity errors. Consider adding validation to handle this scenario.